src/pages/CreateOrderPage.jsx

```diff
@@ -89,8 +89,13 @@ const CreateOrderPage = () => {
   };
 
   const handleDeleteAddress = (addressId) => {
+    if (addressId === selectedAddressId) {
```

Comment:
handleAddNewAddress ve handleSaveEditedAddress işlevlerinde hata durumlarında form gönderimini durduruyorsun, bu doğru bir yaklaşım. Ancak, başarılı işlem sonrasında form alanlarını temizlemek ve başarılı işlemi kullanıcıya bildirmek (örneğin bir toast mesajı ile) kullanıcı deneyimini iyileştirebilir.